js-exercises/user-check-password/user.js

```diff
@@ -0,0 +1,22 @@
+
+// (*)
+// Follow the instructions and fill in the blank sections.
+function User(username, password) {
+  // set a username and password property on the user object that is created
+  let userObj = {};
```

Comment:
Could have been done without creating a new empty object and assigning it to the this keyword. Calling the function with new keyword does all this for you.
eg: `this.username = username;`
      `this.password = password;`